app/schemas.py

```diff
@@ -36,26 +36,33 @@ class Measurement(BaseModel):
         None,
         title="PM1.0",
         description="Concentration of PM1.0 inhalable particles per ug/m3",
+        ge=0,
     )
     pm2dot5: Optional[float] = Field(
         None,
         title="PM2.5",
         description="Concentration of PM2.5 inhalable particles per ug/m3",
+        ge=0,
```

Comment:
According to the datasheet, the max effective value is 500

Reply:
Excellent, is that for the the 3 PMs? I will update the PR with this.

Reply:
It only explicitly specifies it for the PM2.5, but I think it should be valid for the rest too